content/docs/tasks/security/authn-policy/index.md

```diff
@@ -570,6 +570,9 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
 401
 {{< /text >}}
 
+You can also add a JWT policy on istio-ingressgateway. This is often used to define JWT policies that apply at the gateway,
```

Comment:
@liminw that looks good, but I suggest tweaking the sentence a little to make it more clear. Here's my suggestion:

```
You can also add a JWT policy to an ingress gateway (e.g., service `istio-ingressgateway.istio-system`).
This is often used to define a JWT policy for all services bound to the gateway,
```

Reply:
Thanks @frankbu. I updated the doc as you suggested.